config/routes.rb

```diff
@@ -47,5 +48,9 @@
   root :to => 'predictions#home'
 
   match '/healthcheck' => 'content#healthcheck'
+  
+  namespace :api do
+ 	  resources :predictions
```

Comment:
I am confused about why it shows up this way on Github. My editor (Atom) shows no such whitespace.